modules/OCPP201/conversions.cpp

```diff
@@ -864,7 +947,43 @@ to_ocpp_notify_ev_charging_needs_request(const types::iso15118::ChargingNeeds& c
         dc_charging_parameters.evEnergyCapacity = dc.ev_energy_capacity;
         dc_charging_parameters.fullSoC = dc.full_soc;
         dc_charging_parameters.bulkSoC = dc.bulk_soc;
-    }
+    } else if (charging_needs.v2x_charging_parameters.has_value()) {
```

Comment:
is else if correct here, so v2x_charging_parameters can only be set if ac or dc are not set?